yuniql-cli/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Yuniql.CLI": {
       "commandName": "Project",
-      "commandLineArgs": "platforms",
+      "commandLineArgs": "drop --force",

```

Comment:
I think its better we keep the old values. Its dangeours to default this to drop :)